src/installing-deis/installing-deis-workflow.md

```diff
@@ -46,7 +46,8 @@ $ helm fetch deis/deis
 Now that you have Helm installed and added the Deis Chart Repository, install Workflow by running:
 
 ```
-$ helm install deis/deis --namespace=deis
+$ helm generate deis                   # generates various secrets
```

Comment:
Maybe this should be `helm generate -x manifests deis`. That's the current way to work around the double-templating.